File: ru.kmorozov.gbd.db/src/main/java/ru/kmorozov/db/core/logic/model/book/google/BuyLink.java
package ru.kmorozov.db.core.logic.model.book.google;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by km on 28.11.2015.
 */
class BuyLink implements Serializable {

    @SerializedName("Seller")
    private String seller;
    @SerializedName("Url")
    private String url;
    @SerializedName("TrackingUrl")
    private String trackingUrl;
    @SerializedName("IsPublisher")
    private String publisher;
    @SerializedName("Price")
    private String price;
}

File: ru.kmorozov.gbd.db/src/main/java/ru/kmorozov/db/core/logic/model/book/google/BuyLink.kt
package ru.kmorozov.db.core.logic.model.book.google

import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by km on 28.11.2015.
 */
internal class BuyLink : Serializable {

    @SerializedName("Seller")
    private val seller: String? = null
    @SerializedName("Url")
    private val url: String? = null
    @SerializedName("TrackingUrl")
    private val trackingUrl: String? = null
    @SerializedName("IsPublisher")
    private val publisher: String? = null
    @SerializedName("Price")
    private val price: String? = null
}
